fun/maze.cc.lau

```diff
@@ -0,0 +1,27 @@
+{{/*Made by Crenshaw#7860
+Based off Raffael#7777's Carl-Bot command here: https://carl.gg/t/260353
+Command type: prefix/mention
+It also shows the solution, and works with .ExecData*/}}
```

Comment:
This comment is formatted incorrectly. Please refer to the contributing guide for how to format the top comment.